src/test/java/com/softserveinc/ita/rozetka_test/OrderTest.java

```diff
@@ -86,4 +86,16 @@ public void testIncreasingAndDecreasingProductsCountInBasket() {
         assertEquals(decreaseOrderProductSum, increasedOrderProductSum / 2,
                 "The new sum for the product should be twice less than the previous one");
     }
+
+    @Test
+    public void testOpenOrderPage() {
+        var pageTitle = orderPage.getPageTitle();
+        assertTrue(!pageTitle.isEmpty(), "The title shouldn't be empty");
+        assertTrue(pageTitle.equals("Оформлення замовлення"), "The page title should match the condition");
+        var name = "Андрій";
+        var surname = "Гура";
+        orderPage.setName(name).setSurName(surname);
```

Comment:
this is pointless, unless you save or submit data and then check that it's saved or submitted

Reply:
this